client/src/pages/HowToPlay.js

```diff
@@ -0,0 +1,240 @@
+            </Typography>
+            <Typography
+                className={classes.tableOfContentsLink}
+                onClick={(event) => handleChange(event, 7)}
+            >
+                Game Layout
+            </Typography>
+            <Typography
+                className={`${classes.tableOfContentsLink} ${classes.tabbed}`}
+                onClick={(event) => handleChange(event, 8)}
+            >
+                Game Board
+            </Typography>
+            <Typography
+                className={`${classes.tableOfContentsLink} ${classes.tabbed}`}
+                onClick={(event) => handleChange(event, 9)}
+            >
+                Game Chat
+            </Typography>
+            <Typography
+                className={classes.tableOfContentsLink}
+                onClick={(event) => handleChange(event, 10)}
+            >
+                Gameplay
+            </Typography>
+            <Typography
+                className={`${classes.tableOfContentsLink} ${classes.tabbed}`}
+                onClick={(event) => handleChange(event, 11)}
+            >
+                Turn Cycle
+            </Typography>
+            <Typography
+                className={`${classes.tableOfContentsLink} ${classes.tabbed}`}
+                onClick={(event) => handleChange(event, 12)}
+            >
+                Win Conditions
+            </Typography>
+        </React.Fragment>
+    );
+
+    const nextPage = () => {
+        setPage(page + 1);
+    }
+
+    const previousPage = () => {
+        setPage(page - 1);
+    }
+
+    const pages = [pageOne,
+        <Overview />,
+        <GettingStarted />,
+        <CreatingMatch />,
+        <JoingingMatch />,
+        <AssigningRoles />,
+        <GameLayout />,
+        <GameBoard />,
+        <GameChat />,
+        <Gameplay />,
+        <TurnCycle />,
+        <WinConditions />
+    ]
+
+    return (
+        <div className={classes.container}>
+            <Typography color="textPrimary" className={classes.header}>
+                How To Play
+            </Typography>
+            <hr className={classes.hr} />
+            <div className={classes.content}>{pages[page - 1]}</div>
+            <ArrowBackIcon className={page > 1 ? classes.arrowButton : classes.disabled} onClick={page > 1 ? previousPage : null} />
+            <ArrowForwardIcon className={page < 12 ? classes.arrowButton : classes.disabled} onClick={page < 12 ? nextPage : null} />
+            <hr className={classes.hr} />
+            <Pagination
+                count={12}
+                page={page}
+                onChange={handleChange}
+                className={classes.center}
+                color="secondary"
+            />
+            <Button
+                variant="contained"
+                color="secondary"
+                className={classes.button}
+                onClick={returnToHome}
+            >
+                Return To Home
+            </Button>
```

Comment:
This button is floating off the bottom of the card, can you adjust it so it's within?